config/settings/base.py

```diff
@@ -163,13 +164,15 @@
 
 WSGI_APPLICATION = "config.wsgi.application"
 
+LOG_FORMAT_START = "%(asctime)s +0000: thread=%(thread)d"
+LOG_FORMAT_END = f'level=%(levelname)s, app=corgi, env={get_env()}, msg="%(message)s"'
+
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
     "formatters": {
         "default": {
-            "format": "[%(asctime)s] [%(name)s:%(lineno)d] %(levelname)s: %(message)s",
-            "datefmt": "%d/%b/%Y:%H:%M:%S %z",  # Matches the one used by gunicorn
+            "format": f"{LOG_FORMAT_START}, {LOG_FORMAT_END}",
```

Comment:
For this Django logging config, the new version almost has the same information as the old one, just formatted differently and with thread / app / env added. But it's missing name and lineno which I think are useful.

Reply:
Let's add it back later.